src/Components/Web.JS/src/InputFile.ts

```diff
@@ -1,3 +1,4 @@
+import { DotNet } from '@microsoft/dotnet-js-interop';
```

Comment:
Just looking to get thoughts on the general implementation strategy here (using the file streaming API so that we can support large file uploads in Blazor). This is _very_ early stage and there's lots to cleanup 😄 .